.github/workflows/advanced.yml

```diff
@@ -69,16 +69,16 @@ on:
 
 env:
   BINUTILS_REPO: Windows-on-ARM-Experiments/binutils-woarm64
-  BINUTILS_BRANCH: ${{ inputs.binutils_branch || 'woarm64' }}
+  BINUTILS_BRANCH: ${{ inputs.binutils_branch || 'woarm64-v3' }}
```

Comment:
Should I already review changes in these branches?

Reply:
Not yet, it requires a bit more minor adjustments.

Reply:
binutils/gcc branches are validated and ready for the review.